cookbooks/aws-parallelcluster-config/resources/manage_efs.rb

```diff
@@ -14,8 +14,8 @@
 
 property :shared_dir_array, Array, required: true
 property :efs_fs_id_array, Array, required: true
-property :efs_encryption_in_transit_array, Array, required: true
-property :efs_iam_authorization_array, Array, required: true
+property :efs_encryption_in_transit_array, Array, required: false
+property :efs_iam_authorization_array, Array, required: false
```

Comment:
These two properties are optional because they are not needed when unmounting the file system